monkey_island/cc/environment/__init__.py

```diff
@@ -8,7 +9,7 @@ class Environment(object):
     __metaclass__ = abc.ABCMeta
 
     _ISLAND_PORT = 5000
-    _MONGO_URL = "mongodb://localhost:27017/monkeyisland"
+    _MONGO_URL = os.environ.get("MONGO_URL", "mongodb://localhost:27017/monkeyisland")
```

Comment:
MONGO_URL is not specific enough, it might be used for another application.
Update readme.txt accordingly with the new name

```suggestion
    _MONGO_URL = os.environ.get("MONKEY_MONGO_URL", "mongodb://localhost:27017/monkeyisland")
```

Reply:
Changes completed and pushed.